hummingbot/connector/gateway_EVM_AMM.py

```diff
@@ -105,7 +105,10 @@ def __init__(self,
         self._auto_approve_task = None
         self._poll_notifier = None
         self._nonce = None
-        self._native_currency = "ETH"  # make ETH the default asset
+        self._native_currency = "ETH"  # default value only, will be updated in get_chain_info()
```

Comment:
I think we should have `self._native_currency` first initialised as `None`. And only upon the successful assigning of a token in `get_chain_info()` would the connector be deems ready.

i.e. in `status_dict` property, we check that `self._native_currency` is assigned.